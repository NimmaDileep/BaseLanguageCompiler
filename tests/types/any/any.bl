var x: Any = 5;

print(x);

x = "Hello";

print(x);

x = 2;

print(x);

x = false;

print(x);

x = 10;

var y: Int = -5;

x = y;

print(x);
