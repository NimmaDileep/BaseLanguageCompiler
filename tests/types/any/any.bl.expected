5
Hello
2
false
-5
